fboss/agent/platforms/wedge/WedgePlatform.cpp

```diff
@@ -137,7 +137,7 @@ void WedgePlatform::initLocalMac() {
   // For now, we take the MAC address from eth0, and enable the
   // "locally administered" bit.  This MAC should be unique, and it's fine for
   // us to use a locally administered address for now.
-  std::vector<std::string> cmd{"/sbin/ip", "address", "ls", "eth0"};
+  std::vector<std::string> cmd{"/sbin/ip", "address", "ls", "ma1"};
```

Comment:
@sonoble  yeah the plan here is to have you add `--interface ma1` (or similar) to your init scripts - this diff is just to get you able to build... we'll merge the full fixes internally and push